doc/tutorial.txt

```diff
@@ -92,6 +92,11 @@ The category names currently supported are:
 
 Default value is "none".
 
+* ftracedir : String. Prefix used to find tracefs. Defaults to
```

Comment:
Once the "tracing subfolder issue" is fixed, it should probably be simplified into:
ftracedir: String: Mount point of tracefs, or equivalent in older systems. Defaults to "/sys/kernel/debug/tracing"

Reply:
Yep, I tidied that up a bit.